MANUAL_GCP_DEPLOYMENT.md

```diff
@@ -0,0 +1,18 @@
+# Deployment Guide for membrane-backend
+
+This guide outlines the steps to deploy manually membrane-backend to Google Cloud Run.
```

Comment:
when you refer to a name the first time in a document, you need to both spell out the name and its acronym.

such as:

`Google Cloud Run (GCP)`